notebooks/split_labels.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
<p>Long term would it make sense to also just create a csv to avoid interfacing with these jsonl files too much?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>